util/travis/clang-format-whitelist.txt

```diff
@@ -118,6 +118,7 @@ src/inventory.cpp
 src/inventory.h
 src/inventorymanager.cpp
 src/inventorymanager.h
+src/irr_v3d.h
```

Comment:
i tend to prefer to have clang-format comments before and after the problematic section.

Please don't use that and add
```
// clang-format off
<blah>
// clang-format on
```
around the problematic code

Reply:
Please don't, otherwise our code will be filled with a mess of clang on/off lines. The sillyness of LINT shouldn't force us to make a mess of our code, obviously. Please add to whitelist.

Reply:
on a single part it's acceptable for me. We already have such thing sometimes.

Reply:
`clang-format` needs configuration. I don’t mind the style it suggests, generally, but it uses tabs for alignment, treating one as 8 spaces. That’s inherently wrong, tabs can vary in size.

By the way, in an operator overload declaration, it forbids space between the operator symbol and the opening parenthesis. That’s perfectly fine for a function declaration, but operators are different.